Mathlib/NumberTheory/Padics/Hensel.lean

```diff
@@ -303,7 +303,7 @@ private theorem newton_seq_dist_aux (n : ℕ) :
   | k + 1 =>
     have : 2 ^ n ≤ 2 ^ (n + k) := by
       apply pow_le_pow_right
-      norm_num
+      · norm_num
```

Comment:
I'm on the fence whether indenting both goals or just the first one is better. I don't strongly object to your current choice.

Reply:
I ended up indenting this one as well!